docs/reference/content/driver-async/tutorials/authentication.md

```diff
@@ -255,6 +239,14 @@ Or via the `ConnectionString`:
 mongodb://username%40MYREALM.com@myserver/?authMechanism=GSSAPI&authMechanismProperties=SERVICE_NAME:othername
 ```
 
+{{% note %}}
+There are limitations when authenticating from a Windows client. For details, see
```

Comment:
Is there a synopsis of what these bugs mean?

Reply:
Added synopsis